api/client/run.go

```diff
@@ -158,6 +158,7 @@ func (cli *DockerCli) CmdRun(args ...string) error {
 	var (
 		waitDisplayID chan struct{}
 		errCh         chan error
+		exitCh        chan struct{}
```

Comment:
we have to declare `exitCh` out of the `if !config.AttachStdout && !config.AttachStderr` block, since we use it out of this block